nengo/builder/ensemble.py

```diff
@@ -49,6 +49,23 @@ def get_activities(model, ens, eval_points):
         x, model.params[ens].gain, model.params[ens].bias)
 
 
+def get_gain_bias(ens, rng=np.random):
```

Comment:
Why bother extracting this into a function? This refactoring doesn't seem to be being used anywhere else in the pull request. Maybe extract it to a separate one?